### DISCUSSION TITLE
clarify configuration of vs code extension defaults (📜 rome.json)

### DISCUSSION THREAD

**Discussion by probfusion** [rome/tools#4557] [Suggestions]
i think the documentation should clarify how to set defaults for the vs code extension. it took me a while to figure out. 

i wanted single quotes for js, and it worked fine within my project directories with a `rome.json`. but when i wanted to do one-off edits of js files i couldn't figure out how to make rome use single quotes, even with requireConfiguration as `false`. 

---

ℹ️ after some experimentation i found out i could put a `rome.json` in any folder upstream of my workspace folder, and it would be read properly. in addition, a workspace `rome.json` would correctly override the upstream one. ℹ️

---

i'm on windows so i just plopped a `rome.json` in `C:\` and now i have single quotes by default for any workspace in that drive. one-off js edits will still be formatted the way i like.

**Comment by ematipico** [comment]
You're right, our documentation lacks this part of the functionality, and we should update it.

There are few places that need to be updated. Let me know if you want to help update the documentation.
